Validators/CheckinCheckoutValidator.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date
+from typing import Optional
+
+from pydantic import validator
+from pydantic.main import BaseModel
+
+
+class CheckinCheckoutValidator(BaseModel):
+    desired_checkin: Optional[date]
+    desired_checkout: Optional[date]
+
+    @validator('desired_checkin')
+    def date_must_be_greater_than_today(cls, date: Optional[date]) -> Optional[date]:
+        """
+        Check for date to be greater than today
+        @param date: Checkin Date
+        @type date: Optional[date]
+        @return: Optional[date]
+        """
+        if date < date.today():
+            raise ValueError(f'Date must be after today{date.today()}')
+        return date
+
+    @validator('desired_checkout')
+    def checkout_date_present(cls, checkout_date: Optional[date], values) -> Optional[date]:
+        if 'desired_checkin' in values and checkout_date is not None and values['desired_checkin'] is None:
```

Comment:
Break long conditionals up into multilines for readability. Also, I think this is achievable with just 2 conditions instead of 3?
```
if (
   checkout_date is not None and
   values.get('desired_checkin') is None
):
```

Reply:
I think we need to check **'desired checkin' in values**  conditional as well because values contain only the previously validated fields  **values.get('desired_checkin') is None** will throw an error if we don't check the previous condition.

Will make the condition mulitlined.